pkg/cfn/builder/iam_helper.go

```diff
@@ -138,7 +138,7 @@ func createRole(cfnTemplate cfnTemplate, clusterIAMConfig *api.ClusterIAM, iamCo
 		cfnTemplate.attachAllowPolicy("PolicyEFSEC2", refIR, efsEc2Statements())
 	}
 
-	if api.IsEnabled(iamConfig.WithAddonPolicies.AWSLoadBalancerController) {
+	if api.IsEnabled(iamConfig.WithAddonPolicies.AWSLoadBalancerController) || api.IsEnabled(iamConfig.WithAddonPolicies.DeprecatedALBIngress) {
```

Comment:
it adds `PolicyAWSLoadBalancerController` even if `ALBIngress` is set? 🤔 shouldn't it error out and ask to use `AWSLoadBalancerController` explicitly instead?

Reply:
we wanna support both, but log a warning if the user is using `ALBIngress`. Eventually when we remove `ALBIngress` we will want it to error